docs/contributing/naming-convention.md

```diff
@@ -0,0 +1,27 @@
+# Contributing Naming Convention
+
+## General Guidelines
+
+This document outlines the naming conventions for our project. These conventions are intended to promote consistency and clarity across different components.
+
+### Specific Conventions
+
+#### Python Code
+
+  **Style**: We follow the official Python Enhancement Proposal 8 [PEP8]((https://peps.python.org/pep-0008/#package-and-module-names) ) guidelines for naming conventions.
+
+#### Jupyter Notebooks
+
+ **File Names**: File names should start with an underscore (_). Please use underscores instead of hyphens.
```

Comment:
```suggestion
* **File names**: Please use underscores (`_`) instead of hyphens (`-`).
```

I don't think we should have an underscore prefix. Could you share your rationale?

Reply:
Oh, I made a mistake there. I wanted to write exactly what you wrote :)

Reply:
:+1: :)